src/config/dynamic-theme-fixes.config

```diff
@@ -8764,6 +8764,27 @@ section[class$="container"][style^="background-image"] {
 
 ================================
 
+mdigi.tools
+
+CSS
+input {
+    color: #6B6B6B !important;
+}
+label, button, path,
+h1, h2, h3, h4, p, a,
+.site-title {
+    color: #808080 !important;
+}
+[class*="ml-"] {
```

Comment:
This targets a lot of elements, is this really needed as well?

Reply:
> This targets a lot of elements, is this really needed as well?

Yes, all of it is just text.

Reply:
Let me rephrase the question, why is this fix needed?